packages/block-editor/src/components/url-input/index.js

```diff
@@ -139,8 +139,8 @@ class URLInput extends Component {
 		const { showSuggestions, selectedSuggestion, suggestions, loading } = this.state;
 
 		// Trigger `onKeyDown` event, passing the selected suggestion besides the event class.
-		if ( this.props.onKeyDown ) {
-			this.props.onKeyDown( event, selectedSuggestion );
+		if ( isFunction( this.props.__experimentalOnKeyDown ) ) {
```

Comment:
I added `isFunction` check before calling it

Reply:
I think @talldan specifically advised against this because it wouldn't provide useful error reporting.

Reply:
what's your suggested fix? Is `onKeyDown` being completely removed? If so, might be best to just remove it here right away

Reply:
Yeh sorry not overly helpful was I!? 😄 

I believe he was saying to just check if `this.props.__experimentalOnKeyDown` was defined. That way if it wasn't it would give a clearer error. This is where I feel we miss prop types.

```js
if ( this.props.__experimentalOnKeyDown  ) {
```

Reply:
You were - I got it the first time but I also overheard that this property is possibly going away. That's why I'm asking whether I should bother with fixing or maybe remove it too, as we know that's gonna happen anyway.

Reply:
gone